build-system/tasks/firebase.js

```diff
@@ -51,10 +50,10 @@ async function copyAndReplaceUrls(src, dest) {
 async function firebase() {
   if (!argv.nobuild) {
     await clean();
-    if (argv.min) {
-      await dist();
+    if (argv.compiled) {
+      await doDist({fortesting: argv.fortesting});
```

Comment:
I chose to be explicit here because `firebase()` is an entry point for a  separate `gulp` task, and I wanted the code to self-document its usage of `argv`. Leaving this as-is, since the code already has the desired effect.